### PR TITLE
Added Counterspell Manila

### PR DESCRIPTION
**Pull request by romalescarl17**

### CODE REVIEW

**Comment by lachlanjc** [comment]
#### 📝 Changed routes:


| [**/manila**](https://counterspell-m6g7f1d13.hackclub.dev/manila) |
|:-:|
|<a href="https://counterspell-m6g7f1d13.hackclub.dev/manila">Screenshot of /manila</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/xoD4Bn6MvcjoqGPDkQuniLtbRd3xpupRvQSupayS1QP6SNF2l4vvDicMMnEvXkDYPw3sGQjlOAkkNIA32YxQMA.png">(view full size)</a> |







Commit 28c865ee6a8441b45ec2ef110abea4c3c0cd8a49 (https://counterspell-m6g7f1d13.hackclub.dev).

File: components/Manila/Steps.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        <div className="">
+          <p className="py-10 mx-6 text-3xl uppercase retro">
+            How does a <span className="text-pink">game jam</span> work?
+          </p>
+          <div className="flex justify-center mx-12 my-6">
+            <div className="flex flex-wrap items-center justify-center gap-6 p-8 border-4 border-dashed border-pink inter">
+              <StepCard
+                img="/photos/3.png"
+                // TODO: Change `Example City` to the name of your city
+                text="Sign up for Counterspell Example city"
```
Comment (samdev-7) [inline components/Manila/Steps.js:16]:
```suggestion
                text="Sign up for Counterspell Manila"
```

File: components/Manila/Steps.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+              />
+              <StepCard
+                img="/photos/7.png"
+                text="Ship what you made and share it with the rest of the world!"
+              />
+            </div>
+          </div>
+          <div className="py-10 mx-6 space-y-4 text-xl uppercase retro">
+            <p>
+              Counterspell Example City will go on for{" "}
```
Comment (samdev-7) [inline components/Manila/Steps.js:38]:
```suggestion
              Counterspell Manila will go on for{" "}
```

**Review by samdev-7 — CHANGES_REQUESTED** [review]
A few leftover "Example City"s need to be changed. I've left suggestions.

File: components/events.json
Comment (samdev-7) [inline components/events.json:1]: Hi! Can you move this into the Manila folder?

**Review by samdev-7 — CHANGES_REQUESTED** [review]

File: components/events.json
Comment (romalescarl17) [reply components/events.json:1]: Yeah, sure. Please wait :)

Comment (romalescarl17) [reply components/events.json:1]: Moved it :)

**Review by samdev-7 — APPROVED** [review]

**Pull request merged**